libbeat/docs/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -0,0 +1,268 @@
+[role="xpack"]
+[[monitoring-metricbeat-collection]]
+== Collect {beatname_uc} monitoring data with {metricbeat}
+[subs="attributes"]
+++++
+<titleabbrev>Use {metricbeat}</titleabbrev>
+++++
+
+In 7.3 and later, you can use {metricbeat} to collect data about {beatname_uc} 
+and ship it to the monitoring cluster, rather than routing it through the 
+production cluster as described in <<monitoring-internal-collection>>.
+
+ifeval::["{beatname_lc}"=="metricbeat"]
+Because you'll be using {metricbeat} itself to monitor {beatname_uc}, you'll
+need to decide whether you want to run more than one instance of {metricbeat} or
+have {beatname_uc} monitor itself. You have a couple options:
+
+* Run two instances of {beatname_uc}: a main instance that collects metrics
+from the system and services running on the server, and a second instance that
+collects metrics from {beatname_uc} only. Using a separate instance as a
+monitoring agent allows you to send monitoring data to a dedicated monitoring
+cluster. If the main agent goes down, the monitoring agent remains active.
++
+If you're running {beatname_uc} as a service, this approach requires extra work
+because you need to run two instances of the same installed  service
+concurrently.
+
+* Configure your existing {beatname_uc} installation to collect monitoring data
```

Comment:
I wonder if we should document this second approach at all. It's easier to setup but it's not recommended in mission-critical environments (such as a production environment as opposed to a development or test environment) as it prevents the user to ship data to a dedicated monitoring cluster.

Also, with setup is akin to the internal collection approach in that the code that does the monitoring executes in the same process as the code that's being monitored. So if that process dies, you lose not only the monitored Metricbeat but also the monitoring of it.

Reply:
Let's remove it since it's not a best practice.